ic-os/components/ic/generate-replica-config.sh

```diff
@@ -182,55 +135,49 @@ while getopts "l:m:q:n:c:t:i:o:b:" OPT; do
     esac
 done
 
-if [ "${IN_FILE}" == "" -o "${OUT_FILE}" == "" ]; then
+if [ "${CONFIG_FILE}" == "" -o "${IN_FILE}" == "" -o "${OUT_FILE}" == "" ]; then
     usage
     exit 1
 fi
 
 if [ "${NETWORK_CONFIG_FILE}" != "" -a -e "${NETWORK_CONFIG_FILE}" ]; then
     read_network_variables "${NETWORK_CONFIG_FILE}"
 fi
-
-if [ "${BACKUP_CONFIG_FILE}" != "" -a -e "${BACKUP_CONFIG_FILE}" ]; then
-    read_backup_variables "${BACKUP_CONFIG_FILE}"
-fi
-
-if [ "${NNS_CONFIG_FILE}" != "" -a -e "${NNS_CONFIG_FILE}" ]; then
-    read_nns_variables "${NNS_CONFIG_FILE}"
-fi
-
 if [ "${MALICIOUS_BEHAVIOR_CONFIG_FILE}" != "" -a -e "${MALICIOUS_BEHAVIOR_CONFIG_FILE}" ]; then
     read_malicious_behavior_variables "${MALICIOUS_BEHAVIOR_CONFIG_FILE}"
 fi
-
-if [ "${QUERY_STATS_CONFIG_FILE}" != "" -a -e "${QUERY_STATS_CONFIG_FILE}" ]; then
-    read_query_stats_variables "${QUERY_STATS_CONFIG_FILE}"
+if [ "${CONFIG_FILE}" != "" -a -e "${CONFIG_FILE}" ]; then
+    read_config_variables "${CONFIG_FILE}"
 fi
 
-if [ "${JAEGER_ADDR_FILE}" != "" -a -e "${JAEGER_ADDR_FILE}" ]; then
-    read_jaeger_addr_variable "${JAEGER_ADDR_FILE}"
-fi
+
 
 INTERFACE=($(find /sys/class/net -type l -not -lname '*virtual*' -exec basename '{}' ';'))
 IPV6_ADDRESS="${ipv6_address%/*}"
 IPV6_ADDRESS="${IPV6_ADDRESS:-$(get_if_address_retries 6 ${INTERFACE} 12)}"
-IPV4_ADDRESS="${ipv4_address:-}"
+if [[ -n "$ipv4_address" && "$ipv4_address" != "null" && -n "$ipv4_prefix_length" && "$ipv4_prefix_length" != "null" ]]; then
+  IPV4_ADDRESS="${ipv4_address}/${ipv4_prefix_length}"
+else
+  IPV4_ADDRESS=""
+fi
```

Comment:
Before, we were re-setting ipv4_address to contain the prefix_length in build-bootstrap-config-image.sh. Now, we need to do it here, or update the config object directly somewhere in the pipeline.

I hope that in the networking revamp, we can get rid of the prefix length value entirely and just have NPs include the prefix length when they define the address (example: ipv4_address=212.71.126.49/29)